sdk/tests/resources/tex-input-validation.js

```diff
@@ -1,397 +0,0 @@
-/*
```

Comment:
Is this file recreated in the new directory? It looks like it's missing from this pull request.